### PR TITLE
add jquery-requestAnimationFrame v0.1.2

### LINKED ISSUE #6883: [Request] jQuery requestAnimationFrame
**Library name:** jQuery requestAnimationFrame
**Git repository url:** https://github.com/gnarf/jquery-requestAnimationFrame
**License(s):** MIT license
**Official homepage:** https://github.com/gnarf/jquery-requestAnimationFrame

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #6883 
@pvnr0082t  please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/gnarf/jquery-requestAnimationFrame
 * Official website (optional, not the repository): https://github.com/gnarf/jquery-requestAnimationFrame
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 335
   - Count of watchers: 26
   - Count of forks: 35
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

git autoupdate config added, close #6883, cc @gnarf

### CODE REVIEW

File: ajax/libs/jquery-requestAnimationFrame/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "version": "0.1.2",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/gnarf/jquery-requestAnimationFrame.git",
+    "basePath": "dist",
+    "files": [
+      "jquery.requestAnimationFrame.js",
+      "jquery.requestAnimationFrame.min.js"
+    ]
+  },
```
Comment (pvnr0082t) [inline ajax/libs/jquery-requestAnimationFrame/package.json]: Let's use ```**/*``` here.

**Comment by pvnr0082t** [comment]
@kennynaoh Why do I see files of ```jquery-simple-websocket```  here?

**Comment by kennynaoh** [comment]
@pvnr0082t 
Thanks for reviewing my PR
I have changed the package.json
Please review it again
Thanks

File: ajax/libs/jquery-requestAnimationFrame/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/gnarf37/jquery-requestAnimationFrame.git"
+  },
+  "licenses": [
+    {
+      "type": "MIT",
+      "url": "https://github.com/gnarf37/jquery-requestAnimationFrame/blob/master/LICENSE-MIT"
+    }
+  ],
```
Comment (pvnr0082t) [inline ajax/libs/jquery-requestAnimationFrame/package.json]: @kennynaoh  Let's use ```"license": "MIT"``` here

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have updated it.
Please review it again
Thanks

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this pull request is OK but 42 commit behind. Does it need to rebase? Please review it again, thank you.

**Comment by kennynaoh** [comment]
@pvnr0082t 
@PeterDaveHello 
I have rebased it to newest place
Please help me review it again
Thanks

**Comment by PeterDaveHello** [comment]
@kennynaoh try to add some keywords next time.

File: ajax/libs/jquery-requestAnimationFrame/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "target": "git://github.com/gnarf/jquery-requestAnimationFrame.git",
+    "basePath": "dist",
+    "files": [
+      "**/*"
+    ]
+  },
+  "author": {
+    "name": "Corey Frang",
+    "email": "gnarf37@gmail.com",
+    "url": "https://github.com/gnarf37"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-requestAnimationFrame/package.json]: Boom! :boom: 404 not found!

File: ajax/libs/jquery-requestAnimationFrame/package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/gnarf37/jquery-requestAnimationFrame.git"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-requestAnimationFrame/package.json]: Repo url should also be updated.

**Comment by PeterDaveHello** [comment]
@kennynaoh please send another PR to fix it.

**Comment by PeterDaveHello** [comment]
ping @kennynaoh

**Pull request merged**